### PR TITLE
Room list & room detail [Client]

### PR DESCRIPTION
**Pull request by phamviethieu**

### CODE REVIEW

**Comment by phamviethieu** [comment]
Peek 2020-06-30 16-49
ready

**Comment by phamviethieu** [comment]
ready

File: resources/views/functions/room_detail.blade.php [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+                    </div>
+
+                    @if (Auth::check())
+                        <div class="contact-1 sidebar-widget">
+                            <div class="main-title-2">
+                                <h1>{{ trans('message.functions.leave_a_comment') }}</h1>
+                            </div>
+                            <div class="contact-form">
+                                <form id="contact_form" action="" method="POST"
+                                      enctype="multipart/form-data">
```
Comment (huyenttm-0901) [inline resources/views/functions/room_detail.blade.php]:
```suggestion
                            <div class="contact-form">
                                <form id="contact_form" action="" method="POST"
                                    enctype="multipart/form-data">
```

File: resources/views/functions/room_detail.blade.php [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+                                                        @csrf
+                                                        <button type="submit" id="btn-submit" class="btn btn-danger delete">
+                                                            {{ trans('message.functions.delete') }}
+                                                        </button>
+                                                    </form>
+                                                    @endcan
+                                                    </p>
+
+                                            </div>
+                                        </div>
```
Comment (huyenttm-0901) [inline resources/views/functions/room_detail.blade.php]: indent

File: routes/web.php [hunk 3]
```diff
@@ -18,11 +18,14 @@
...
-        Route::post('update', 'UserController@update')->name('user.update');
-        Route::post('cancel', 'UserController@cancelBooking')->name('user.cancel_booking');
+    Route::group(['namespace' => 'Client'], function () {
+        Route::group(['prefix' => 'user'], function () {
+            Route::get('/', 'UserController@index')->name('user.index');
+            Route::get('edit', 'UserController@edit')->name('user.edit');
+            Route::post('update', 'UserController@update')->name('user.update');
+            Route::post('cancel', 'UserController@cancelBooking')->name('user.cancel_booking');
+        });
+        Route::resource('room', 'RoomController');
```
Comment (huyenttm-0901) [inline routes/web.php]:
```suggestion
        Route::resource('rooms', 'RoomController');
```

**Comment by phamviethieu** [comment]
ready

**Pull request merged**